### PR TITLE
Add Contributing page

### PR DESCRIPTION
**Pull request by tombruijn**
First setup of a Contributing page, as referenced by the README since: https://github.com/appsignal/appsignal-ruby/pull/197

I thought I would set up a small page for starters, but it grew into this slightly bigger page.

Feel free to add any changes directly instead of a review.

### CODE REVIEW

File: source/contributing/index.html.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+issue on the project's GitHub page.
+
+It's also possible to submit a feature on one of our Open Source projects by
+creating a Pull Request targeted on the project's `develop` branch.
+
+## Security issues
+
+If you think you've found a security issue with regards to our application,
+network or integrations, please let us know immediately by sending an email to
+[security@appsignal.com](mailto:security@appsignal.com).
```
Comment (matsimitsu) [inline source/contributing/index.html.md]: Just to be absolutely sure, this address exists, @thijsc ?

File: source/contributing/index.html.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+---
+
+Interested in contributing to AppSignal? Great!
+
+Helping out is possible in many ways. By reporting issues, fixing bugs, adding
+features and even fixing typos. [Let us know][contact] if you have any
+questions about contributing to any of our projects.
+
+We're very happy sending anyone who contributes Stroopwafles. Have look at
+everyone we send a package to so far on our [Stroopwafles page][waffles-page].
```
Comment (jeffkreeftmeijer) [inline source/contributing/index.html.md]: > ... everyone we send a package ... 

"sent" instead of "send":

> ... everyone we sent a package ...

Comment (tombruijn) [inline source/contributing/index.html.md]: I blame copy-paste 😅

File: source/appsignal/contributing.html.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+title: Contributing
+---
+
+Interested in contributing to AppSignal? Great!
+
+Helping out is possible in many ways. By reporting issues, fixing bugs, adding
+features and even fixing typos. [Let us know][contact] if you have any
+questions about contributing to any of our projects.
+
+We're very happy sending anyone who contributes Stroopwafles. Have look at
```
Comment (wesoudshoorn) [inline source/appsignal/contributing.html.md]: stroopwaffles or stroopwafels

File: source/appsignal/contributing.html.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+be found on every project's GitHub page under "Releases".
+
+## Bugs
+
+Report a bug by opening an issue on the GitHub project page of the respective
+project. If the bug report contains some sensitive information that is
+necessary for the complete report you can also [contact us][contact] to submit
+the bug.
+
+When the bug is a security sensitive issue, please refer to "Security issues"
```
Comment (wesoudshoorn) [inline source/appsignal/contributing.html.md]: add deeplink?

File: source/appsignal/contributing.html.md (hunk 3 above)
Comment (tombruijn) [inline source/appsignal/contributing.html.md]: Added them to my dictionary for spellchecking 👍

**Comment by tombruijn** [comment]
Updated. Please re-review

**Comment by tombruijn** [comment]
Merging this so we don't have dead urls from the Ruby gem README anymore

**Pull request merged**